### DISCUSSION TITLE
Libmesh for newapp

### DISCUSSION THREAD

**Discussion by mdh23rct** [idaholab/moose#25731] [Q&A Modules: Phase field]
Dear all,
I am generating a newapp from scratch using the MOOSE phase field module. 
Next, I have to integrate libmesh with the newapp. Since I am entirely new in this field, would you be kind to advise me, how I can build libmesh for a newapp, and integrate it with the app executable!

I appreciate your time.

Best regards.

**Comment by GiudGiud** [comment]
Hello

Please follow the application development tutorial. The instructions on how to create an app show you have to use the stork script for that 

Guillaume

**Reply by lindsayad** [reply]
libmesh is a dependency of MOOSE. It is already integrated in any MOOSE application
